Makefile

```diff
@@ -87,9 +88,13 @@ clean:
 	rm -rf cover.out .cover/ cover.html lint.log fmt.log
 
 .PHONY: test
-test: go-gen
+test: go-gen test-otel
 	bash -c "set -e; set -o pipefail; $(GOTEST) ./... | $(COLORIZE)"
 
+.PHONY: test-otel
+test-otel:
+	cd ${OTEL_COLLECTOR_DIR} && bash -c "set -e; set -o pipefail; $(GOTEST) ./... | $(COLORIZE)"
```

Comment:
tests in `cmd/opentelemetry-collector` are excluded bc it is a separate go module. I have  found only this workaround.

It prints this when `go test ./..` is executed from the root dir

```
go: directory cmd/opentelemetry-collector is outside main module
go: directory cmd/opentelemetry-collector/app/defaults is outside main module
go: directory cmd/opentelemetry-collector/app/exporter is outside main module
go: directory cmd/opentelemetry-collector/app/exporter/kafka is outside main mo
```